spec/models/group_spec.rb

```diff
@@ -689,7 +653,60 @@ def can_view?(user, group)
 
   end
 
+  describe '#remove' do
+    before { group.add(user) }
+
+    context 'when group has a title' do
+      before { group.update(title: 'Awesome') }
+
+      it "does not strip user's title if not exact match" do
+        user.update_columns(title: 'Different')
+        expect { group.remove(user) }.to_not change { user.reload.title }
```

Comment:
one downside of this reworking is that over time it makes the test suite slower.

If we can do 2 assertions after the change it is cheaper than repeating the group creation, adding user, removing user N times. 

Can you rework this change overall to avoid large amounts of non-reused db work?